elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -72,6 +72,7 @@ $LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, (
   padding: pfe-local(Padding) !important;
   color: pfe-local(Color) !important;
   font-family: pfe-local(FontFamily);
+  --pf-c--FontSize: #{pfe-local(FontSize)};
```

Comment:
Can we list CSS vars at the top of the list just as an organizational technique?

Reply:
Do you mean just move this CSS var up above the other properties?